<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReapprovisionnerStockDialog</class>
 <widget class="QDialog" name="ReapprovisionnerStockDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>[nom du produit]: reapprovisionner le stock</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="productNameStaticLabel">
       <property name="font">
        <font>
         <family>Outfit</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Produit:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_nom_produit">
       <property name="font">
        <font>
         <family>Outfit</family>
         <pointsize>10</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>[nom du produit]</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_quantite">
       <property name="font">
        <font>
         <family>Outfit</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Quantité à ajouter</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="spin_box_quantite_ajouter">
       <property name="font">
        <font>
         <family>Outfit</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="maximum">
        <double>999999999.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_unite">
       <property name="font">
        <font>
         <family>Outfit</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Unité (base):</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_unite_de_base">
       <property name="font">
        <font>
         <family>Outfit</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>[Unité de base]</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="spacer_vertical">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>71</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="spacer_horizontal">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="bouton_annuler">
       <property name="font">
        <font>
         <family>Outfit</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {background-color:   rgb(217, 217, 217);  
padding: 5px 10px; border: none; border-radius: 4px;}</string>
       </property>
       <property name="text">
        <string>Annuler</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="bouton_ok">
       <property name="font">
        <font>
         <family>Outfit</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {background-color:  rgb(87, 182, 255);  
padding: 5px 10px; border: none; border-radius: 4px;}</string>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
